subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/java/tasks/CreateDependenciesFile.java

```diff
@@ -0,0 +1,61 @@
+package org.shipkit.internal.gradle.java.tasks;
+
+import java.util.Arrays;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.artifacts.ModuleVersionIdentifier;
+import org.gradle.api.artifacts.ResolvedArtifact;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.OutputFile;
+import org.gradle.api.tasks.TaskAction;
+import org.shipkit.internal.gradle.util.StringUtil;
+import org.shipkit.internal.notes.util.IOUtil;
+
+import java.io.File;
+
+public class CreateDependenciesFile extends DefaultTask {
+
+    @Input
+    private Configuration configuration;
+    @OutputFile
+    private File outputFile;
+
+    @TaskAction public void createFile() {
```

Comment:
Might be cool to add information about the content at the top of the file. Perhaps a line that starts with '#' indicating a user comment. Something like:

```
# This file was generated by Shipkit Gradle plugin. See http://..."
```